rust/vectordb/src/table.rs

```diff
@@ -43,7 +43,7 @@ impl std::fmt::Display for Table {
 
 #[derive(Default)]
 pub struct OpenTableParams {
-    pub(crate) dataset_read_params: ReadParams,
+    pub dataset_read_params: ReadParams,
```

Comment:
do we want to make name consistent between "dataset_read_params" or `open_table_params`?